### ISSUE TITLE
Consider changing the name back to Barnold

### ISSUE THREAD

**Issue by jetveam** [tyler-furby/Arnold-For-Blender#20]
A bit of an odd request, but I (and some others) believe the previous name was a lot more distinctive.

**Comment by melMass** [comment]
> A bit of an odd request, but I (and some others) believe the previous name was a lot more distinctive.

I'm not sure of the status of this project, it's supposed to be released for years, I stumbled upon this one yesterday: 

https://github.com/lunadigital/btoa

It works in Blender 3.0 and Arnold 7.0.0.1

**Comment by 4minus1d** [comment]
I agree with jetveam and would like to make this request as well. Please consider changing the name back to **Barnold™**. 

Thank you : )
